vision-zero

```diff
@@ -187,9 +175,26 @@ def replicateDb(args):
             print("This can take up to 15 minutes.")
         print(" ".join(replicate_runner_command) + " " + " ".join(replicate_command))
         snapshot = open(snapshotFQP, "w")
-        subprocess.run(replicate_runner_command + replicate_command, stdout=snapshot)
+
+        try:
+            result = subprocess.run(
+                replicate_runner_command + replicate_command,
+                stdout=snapshot,
+            )
+        except Exception as e:
+            os.remove(snapshotFQP)  # Remove the incomplete file
```

Comment:
removing the file prevents us from entering the doom loop of `replicate-db` sourcing the invalid snapshot file instead of fetching a new one, but it also prevents us from inspecting that file for error messages. i guess removing the snapshot is not idea? 🤔